tests/nowdoc/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -929,8 +929,7 @@ string
 string
 string
 EOD
-    ,
-    $var
+    , $var
```

Comment:
Why it is changed?

Reply:
I think this is more of a side effect, the main idea of the PR was to fix #1292. It doesn't look too bad though, does it?

Reply:
@evilebottnawi what do you say, good to go? This is now literally the last open issue for our release :wink:

Reply:
I think will be better to put it on new line, because we always put each argument on next line

Reply:
Think we can do that in a separate PR?

Reply:
Yes, just open a new issue

Reply:
done! I'll make the release now :tada: